CMakeLists.txt

```diff
@@ -753,6 +753,27 @@ if(swift_build_android AND NOT "${SWIFT_ANDROID_NDK_PATH}" STREQUAL "")
   endif()
 endif()
 
+# Should we cross-compile the standard library for Fuchsia
+is_sdk_requested(FUCHSIA swift_build_fuchsia)
+if(swift_build_fuchsia AND NOT "${SWIFT_FUCHSIA_BUILD_PATH}" STREQUAL "")
+  # HACK: support arm64 fuchsia builds
+  set(SWIFT_FUCHSIA_SYSROOT
```

Comment:
Can we please make this settable externally instead of hardcoding the path within Fuchsia checkout? The name should also contain architecture so you can add aarch64 in the future (e.g. in Clang we [expect](https://fuchsia.googlesource.com/third_party/clang/+/master/cmake/caches/Fuchsia.cmake#42) `FUCHSIA_x86_64_SYSROOT` and `FUCHSIA_aarch64_SYSROOT` to be set).

Reply:
Yeah I can do that. I will also need have a setting for specifying the shared libs directory from Fuchsia since I need to link ICU (given that it's outside of the zircon generated sysroot). Originally I had it so that you just passed just the sysroot for Fuchsia into the build-util but I was doing really brittle path transversal to our "x86-shared" folder so that I could find and link ICU libs. 

Need to think of a clean way to do that on the built-util args so I'm not adding a new arg for each arch.

Reply:
Fixed in the latest build. Removed all hardcoded paths.

You can now set everything as args:
```
utils/build-script --fuchsia -j500 \
--extra-stdlib-deployment-targets=fuchsia-aarch64,fuchsia-x86_64 \
--fuchsia-toolchain-path $FUCHSIA_DIR/buildtools/linux-x64/clang \
--fuchsia-icu-uc-include $FUCHSIA_DIR/third_party/icu/source/common \
--fuchsia-icu-i18n-include $FUCHSIA_DIR/third_party/icu/source/i18n \
--fuchsia-x86_64-sysroot $FUCHSIA_DIR/out/build-zircon/build-zircon-pc-x86-64/sysroot \
--fuchsia-aarch64-sysroot $FUCHSIA_DIR/out/build-zircon/build-zircon-qemu-arm64/sysroot \
--fuchsia-x86_64-libs $FUCHSIA_DIR/out/debug-x86-64/x64-shared \
--fuchsia-aarch64-libs $FUCHSIA_DIR/out/debug-aarch64/arm64-shared  \
--export-compile-commands true --lld --build-runtime-with-host-compiler true \
--skip-test-fuchsia-host --skip-test-linux --build-swift-static-stdlib true \
--build-swift-dynamic-sdk-overlay true
```